CMakeLists.txt

```diff
@@ -748,9 +748,9 @@ set(TIC80STUDIO_SRC
     ${TIC80LIB_DIR}/ext/png.c
 )
 
-if(${BUILD_PRO})
+#if(${BUILD_PRO})
     set(TIC80STUDIO_SRC ${TIC80STUDIO_SRC} ${TIC80LIB_DIR}/studio/project.c)
```

Comment:
This is needed because we need `tic_project_ext` unless you wanted to factor that out into `tools`... I'm not sure what the harm is in building this file though - only the single function should actually get linked during compile.